generic/proof-script.el

```diff
@@ -1486,6 +1486,7 @@ Besides stuff that is not yet documented here, this function
   and sections.
 - enters some commands and their spans in some database (with for
   me unknown purpose)"
+  (reset-goals-modeline) ; Erase goal buffer when a proof is closed
```

Comment:
The comment here is inconsistent with your documentation of reset-goals-modeline.

Reply:
Maybe this call does not break other assistants, but it should not be called for all assistants. Please check if you can find a suitable hook to add this call.